content/posts/newsletter/2019-12-03.md

```diff
@@ -0,0 +1,135 @@
+---
+title: XMPP in all languages! 03 Dec 2019
+save_as: 2019/12/newsletter-03-december/index.html
+url: 2019/12/newsletter-03-december/
+date: 2019-12-03 15:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+## Intro 
+
+Welcome to the XMPP newsletter covering the month of November 2019.
+
+Help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+Edivaldo Brito has written in Portugese "[Como instalar o moderno cliente Jabber/XMPP Dino no Linux via Snap](https://www.edivaldobrito.com.br/cliente-jabber-xmpp-dino-no-linux/)" ("How to install the modern Jabber/XMPP Dino client on Linux via Snap").
+
+Last month, the XMPP newsletter has been translated by community members:
+* French: [Lettre d’information XMPP, 8 novembre 2019, sprints, IoT et le début de Twitter](https://linuxfr.org/news/lettre-d-information-xmpp-8-novembre-2019-sprints-iot-et-le-debut-de-twitter)
+* German: [XMPP Newsletter, 08. November 2019, Sprints, IoT und die Anfänge von Twitter](https://www.jabber.de/xmpp-newsletter-08-november-2019-sprints-iot-und-die-anfaenge-von-twitter/)
+* Spanish: [XMPP Newsletter, 08 Nov 2019, Sprints, IoT, y los primeros Twitter](https://www.jabber.de/es-xmpp-newsletter-08-nov-2019-sprints-iot-y-los-primeros-twitter/)
+
+
+We encourage you all to contribute to this translation effort!
```

Comment:
Maybe: "We are extremely grateful for these contributions! If you can contribute a translation in your own language, please contact <someone>"

Reply:
"We are extremely grateful for these contributions! If you can contribute a translation in your own language, please [contact the CommTeam](https://xmpp.org/about/xsf/comm-team.html)."
?